cocli/cmd/cotsCreateCorim.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
```

Comment:
As discussed with Sabreen and thomas, this file is not needed to create Cots based CoRIM

the existing CoRIM Create command should take CoTSID similar to CoMID and CoSWID.